omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -988,10 +1004,7 @@ def _move_hand_joint(self, joint_pos):
         """
         with PlanningContext(self.env, self.robot, self.robot_copy, "original") as context:
             plan = plan_arm_motion(
-                robot=self.robot,
-                end_conf=joint_pos,
-                context=context,
-                torso_fixed=m.TIAGO_TORSO_FIXED,
+                robot=self.robot, end_conf=joint_pos, context=context, torso_fixed=m.TIAGO_TORSO_FIXED, verbose=True
```

Comment:
Do you actually want it to be verbose here? I thought you only wanted to see the messages during execution? Also this doesnt actually affect planning since you dont pass verbose into OMPL. Leave unresolved, lets discuss.